tools/vim/.ycm_extra_conf.py

```diff
@@ -0,0 +1,106 @@
+# A modified example from YCM main repository
```

Comment:
Here is the problem: I've modified the sample for our purposes. The original license is: http://unlicense.org/. I assume that I can just publish it under HPX's standard Boost licence?